CONTENTS.md

```diff
@@ -0,0 +1,423 @@
+* Documentation: --
+* Package: openSUSE-release-tools-staging-bot
+* Usage: gocd
+
+#### legal-auto
+
+Makes automatic legal reviews based on the legaldb API
+
+* Sources: [legal-auto.py](legal-auto.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: gocd
+
+#### check_tags_in_requests
+
+Checks that a submit request has corrrect tags specified.
+
+* Sources: [check_tags_in_requests.py](check_tags_in_requests.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: gocd
+
+#### abichecker
+
+Checks ABI compatibility in OBS requests.
+
+* Sources: [abichecker](abichecker)
+* Documentation: --
+* Package: openSUSE-release-tools-abichecker
+* Usage: gocd
+
+#### check_source_in_factory
+
+Checks if the sources of a submission are either in Factory or a request for Factory with the same
+sources exist.
+
+* Sources: [check_source_in_factory.py](check_source_in_factory.py)
+* Documentation: [docs/factory-source.asciidoc](docs/factory-source.asciidoc)
+* Package: openSUSE-release-tools
+* Usage: gocd?
+
+#### openqa-maintenance
+
+OpenQA stuff, not sure about the details.
+
+* Sources: [openqa-maintenance.py](openqa-maintenance.py) and [oqamaint](uqamaint)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: gocd?
+
+#### check_tags_in_requests.py
+
+Checks that a submit request has corrrect tags specified.
+
+* Sources: [check_tags_in_requests.py](check_tags_in_requests.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: gocd?
+
+#### repo-checker
+
+Inspects built RPMs from staging projects.
+
+* Sources: [project-installcheck.py](project-installcheck.py),
+  [staging-installcheck.py](staging-installcheck.py),
+  [maintenance-installcheck.py](maintenance-installcheck.py),
+  [findfileconflicts](findfileconflicts), [write_repo_susetags_file.pl](write_repo_susetags_file.pl)
+* Documentation: --
+* Package: openSUSE-release-tools-repo-checker
+* Usage: gocd?
+
+#### manager_42.py
+
+Maintains `00Meta/lookup.yml`.
+
+* Sources: [manager_42.py](manager_42.py)
+* Documentation: --
+* Package: openSUSE-release-tools-leaper
+* Usage: Obsoleted by origin-manager?
+
+### OSC Plugins
+
+#### osc-check_source.py
+
+Checks for usual mistakes and problems in the source packages submitted by users. Used also as
+review bot that assigns reviews (?).
+
+* Sources: [check_source.py](check_source.py) and [check_source.pl](check_source.pl)
+* Documentation: [docs/check_source.asciidoc](docs/check_source.asciidoc)
+* Package: openSUSE-release-tools-check-source
+* Usage: gocd?
+
+#### osc-origin.py
+
+Tools for working with origin information.
+
+* Sources: [osc-origin.py](osc-origin.py)
+* Documentation: [docs/origin-manager.md](docs/origin-manager.py)
+* Package: openSUSE-release-tools-origin-manager
+* Usage: gocd?
+
+#### osc-cycle.py
+
+Helps with OBS cycles visualization.
```

Comment:
what are OBS cycles? google does not help much. Is it build dependency cycles?

Reply:
https://build.opensuse.org/project/repository_state/openSUSE:Factory/stand

Reply:
@coolo I am getting 404. Do I need specific permissions?

Reply:
Fixed URL

Reply:
thanks, so it is build dependency cycles. I  would specify it here, so it is clear which cycles are meant and adding the URL @coolo mentioned would be also helpful as usage example.